### ISSUE TITLE
Instructions for Dolphin support with KDE 5

### ISSUE THREAD

**Issue by gregorywilliams** [rabbitvcs/rabbitvcs#290]
Just to note that the instructions here for Dolphin support for KDE 4.x don't work with KDE 5:

https://github.com/rabbitvcs/rabbitvcs/blob/master/clients/dolphin/README

For KDE 5 the files instead need to be copied to:

/usr/share/kservices5

**Comment by metal450** [comment]
Even with the updated path, it doesn't seem to work: there are no icon overlays, and only menus for SVN (not Git).

Just moved over from Nemo, where it was working great. Were you actually able to get icons & git working in Dolphin?

**Comment by gregorywilliams** [comment]
I see this top-level menu:
![image](https://user-images.githubusercontent.com/1443772/81465225-59872980-91c0-11ea-952d-c773403a9cd2.png)
So, agreed, no Git. Since I've been using RabbitVCS just for SVN so far I hadn't thought of that, though.
These are the icons I see on the **RabbitSVN (SVN)** sub-menu:
![image](https://user-images.githubusercontent.com/1443772/81465257-ac60e100-91c0-11ea-9148-13729d274789.png)
I'm not sure which overlays I should expect to see on the icons.

**Comment by gregorywilliams** [comment]
Back on the top-level menu, I notice that there's not an icon for the **RabbitVCS (SVN)** item on that, though.

**Comment by metal450** [comment]
No, not menu icons. Overlay icons.  i.e. see the screenshots: http://rabbitvcs.org. 

Or https://github.com/rabbitvcs/rabbitvcs/issues/224.

Or https://github.com/rabbitvcs/rabbitvcs/issues/272

**Comment by gregorywilliams** [comment]
No, I don't see overlay icons on the files / directories within Dolphin:
![image](https://user-images.githubusercontent.com/1443772/81465561-c3a0ce00-91c2-11ea-86e5-1a2939f4ebe0.png)
![image](https://user-images.githubusercontent.com/1443772/81465580-e7641400-91c2-11ea-848a-8a1415d018f4.png)

**Comment by metal450** [comment]
Yeah, bummer.  So I guess they don't have integration with Dolphin :(

**Comment by gregorywilliams** [comment]
It doesn't look like it even attempts to handle the overlay icons; the files are just for submenus. The originally-mentioned README does mention that it's experimental Dolphin/KDE support. This issue was purely mentioning that the README could be brought up-to-date to cover the KDE5 case too.

**Comment by metal450** [comment]
Yup, gotcha. Was just hopeful when I saw the name of the issue, but looks like back to Nemo I go :)

**Comment by gregorywilliams** [comment]
The menu items themselves are functional, though. So, if you're using SVN and don't specifically need to see the overlay icons, then it's usable with Dolphin.

**Comment by metal450** [comment]
Personally, the overlays are my primary usage (I have git repos all over the place, & find it extremely handy to be able to easily see/notice which folders are & aren't, and their statuses).

**Comment by monnerat** [comment]
Thanks for the report and info.
Commit 2779686 updates the dolphin README for KDE 5.
Please remember this extension is experimental and no active developer uses it.

As this extension only consists in menu entries calling the cli client, (this is not a real dolphin plugin) You wont have emblems added in the main window.

**Comment by monnerat** [comment]
Dolphin menus now supports Git and french translation. See 40bb6bf

**Comment by metal450** [comment]
Missing from the Git menus:
*Clone
*Revert

**Comment by ddobrev** [comment]
@monnerat the instructions work immediately for the menus. However, I don't get any overlays. Are they supported?

**Comment by metal450** [comment]
@ddobrev Overlays are already discussed above multiple times (including monnerat stating that they are not supported).

**Comment by ddobrev** [comment]
Just saw it, sorry. Thank you for the pointer.

**Comment by monnerat** [comment]
> the instructions work immediately for the menus. However, I don't get any overlays. Are they supported?

Maybe you could read https://github.com/rabbitvcs/rabbitvcs/blob/master/clients/dolphin/README first, then https://github.com/rabbitvcs/rabbitvcs/issues/290#issuecomment-637684283

**Comment by ddobrev** [comment]
I did but I thought "there is no integration with the Dophin main window interface" meant something else. Overlays aren't mentioned in that document.
